pennylane/devices/default_gaussian.py

```diff
@@ -733,7 +733,7 @@ def apply(self, operation, wires, par):
             # set the new device state
             mu, cov = self._operation_map[operation](*par, hbar=self.hbar)
             # state preparations only act on at most 1 subsystem
-            self._state = set_state(self._state, device_wires[0], mu, cov)
+            self._state = set_state(self._state, device_wires[:1], mu, cov)
```

Comment:
This is to ensure that the returned wires remains a list?

Reply:
Yes, a `Wires` instance to be more precise.